app/presenters/wpcc/period_percents_mapper_presenter.rb

```diff
@@ -1,7 +1,7 @@
 module Wpcc
-  class PeriodFilterPresenter < Presenter
+  class PeriodPercentsMapperPresenter < Presenter
     def contribution_percents_explanation
-      @last_period = periods.last
+      @period = map
```

Comment:
This is kind of weird. I understand the logic but I wonder if someone coming fresh to this would get it?

Reply:
Yes, I have thought the same and wondered whether I was being too picky. Will change it to something more descriptive

Reply:
Ok, not going to touch it as per Malcolm's comment above